experimental/dds/tree2/.dependency-cruiser-known-violations.json

```diff
@@ -510,5 +510,40 @@
 			"src/feature-libraries/editable-tree/editableField.ts",
 			"src/feature-libraries/editable-tree-2/index.ts"
 		]
+	},
+	{
+		"type": "cycle",
+		"from": "src/feature-libraries/editable-tree-2/proxies/index.ts",
+		"to": "src/feature-libraries/editable-tree-2/proxies/proxies.ts",
+		"rule": {
+			"severity": "error",
+			"name": "no-circular"
+		},
+		"cycle": [
+			"src/feature-libraries/editable-tree-2/proxies/index.ts",
+			"src/feature-libraries/editable-tree-2/proxies/proxies.ts",
+			"src/feature-libraries/contextuallyTyped.ts",
```

Comment:
Note the "contextuallyTyped" dep in each of these new cycles. It means they are probably not actually new cycles, but are symptoms of the existing cycle between editable tree 1 and editable tree 2. I did play around a bit to see if I could avoid these, to no avail.